kuflow-samples-temporal-uivision-spring/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021-present KuFlow S.L.
+  ~
+  ~ All rights reserved.
+  -->
+<configuration scan="true">
+  <property name="LOG_FILE" value="${LOG_FILE:-${LOG_PATH:-${LOG_TEMP:-${java.io.tmpdir:-/tmp}}}/kuflow-engine-samples-worker-loan.log}"/>
+
+  <include resource="org/springframework/boot/logging/logback/base.xml" />
+
+  <logger name="javax.activation" level="WARN" />
```

Comment:
Maybe we must clean these files, here we have too many 'logger' entries, and some are related to some technologies that we don't have in the repo.

Reply:
ok